pyproject.toml

```
@@ -52,18 +52,15 @@ jaxopt = "0.5.5"
 seaborn = "0.10.1"
 kahypar = "1.1.7"
 openfermionpyscf = "0.5"
-covalent = "0.202.0.post1"
+covalent = "0.227.0rc0"
 pyzx = "0.7.0"
 neural-tangents = "0.6.2"
 plotly = ">=4.5.0"
 optax = "0.1.3"
+mitiq = "0.29.0"
```

asking mostly bc I don't know our general approach to deps in the qml repo but - any reason to keep it hard-pinned?
```suggestion
mitiq = ">=0.29.0"
```